packages/utils/popup-manager.ts

```diff
@@ -65,10 +67,16 @@ const getModal = function (): HTMLElement {
 
 const instances = {}
 
-const PopupManager: IPopupManager = {
+export const PopupManager: PopupManager = {
   modalFade: true,
   modalDom: undefined,
-  zIndex,
+  globalInitialZIndex: 2000,
+  zIndex: 0,
+
+  getInitialZIndex() {
+    if (!getCurrentInstance()) return this.globalInitialZIndex
+    return useGlobalConfig('zIndex').value ?? this.globalInitialZIndex
```

Comment:
```suggestion
let zIndex = this.globalInitialZIndex
if (getCurrentInstance()) {
  zIndex = useGlobalConfig('zIndex').value ?? zIndex
}

return zIndex
```

Reply:
Why？

Reply:
1. `useGlobalConfig` can only be used when `getCurrentInstance` returns the current rendering instance, per your code
2. For readability's sake, this function body is only for like less than 5 lines, we don't really need to use such brief way to write the code.

Reply:
I agree.